test/test_maskedtensor.py

```diff
@@ -64,13 +64,9 @@ def _compare_mts(mt1, mt2, rtol=1e-05, atol=1e-08):
     if not _tensors_match(a, b, exact=False, rtol=rtol, atol=atol):
         raise ValueError("The data in MaskedTensor mt1 and MaskedTensor mt2 do not match")
 
-def _make_tensor_mask(shape, device):
-    return make_tensor(
-        shape, device=device, dtype=torch.bool, low=0, high=1, requires_grad=False
-    )
 
 def _create_random_mask(shape, device):
-    return torch.randint(0, 2, shape, device=device).bool()
```

Comment:
`make_tensor` uses `low=0, high=2` by default for `dtype=torch.bool` and thus we don't have to specify it.